src/utils/viewer/viewer.cpp

```diff
@@ -523,15 +523,14 @@ Viewer::Viewer(const Arguments& arguments)
        esp::agent::ActionSpec::create(
            "lookDown", esp::agent::ActuationMap{{"amount", lookSensitivity}})},
   };
-  agentConfig.sensorSpecifications[0]->resolution =
-      esp::vec2i(viewportSize[1], viewportSize[0]);
+  auto pinholeCameraSpec = esp::sensor::CameraSensorSpec::create();
+  pinholeCameraSpec->sensorSubType = esp::sensor::SensorSubType::Pinhole;
+  pinholeCameraSpec->sensorType = esp::sensor::SensorType::Color;
+  pinholeCameraSpec->position = {0.0f, 1.5f, 0.0f};
+  pinholeCameraSpec->orientation = {0, 0, 0};
+  pinholeCameraSpec->resolution = esp::vec2i(viewportSize[1], viewportSize[0]);
+  agentConfig.sensorSpecifications = {pinholeCameraSpec};
 
-  agentConfig.sensorSpecifications[0]->sensorSubType =
-      args.isSet("orthographic") ? esp::sensor::SensorSubType::Orthographic
```

Comment:
@vauduong :
This is something we did not capture.
You should set the the sub-type based on the input argument.
Please issue any other quick fix when possible! Thanks!

Rename the `pinholeCameraSpec` to `cameraSpec` since it could be ortho camera as well.

Reply:
Ok, I'll make that fix!